pyiceberg/catalog/dynamodb.py

```diff
@@ -393,7 +393,7 @@ def drop_namespace(self, namespace: Union[str, Identifier]) -> None:
             raise NoSuchNamespaceError(f"Database does not exist: {database_name}") from e
 
     def list_tables(self, namespace: Union[str, Identifier]) -> List[Identifier]:
-        """List tables under the given namespace in the catalog (including non-Iceberg tables).
+        """List Iceberg tables under the given namespace in the catalog.
```

Comment:
I noticed that we only updated the docstring here, without a change in the code.
Does Dynamodb already only return Iceberg tables?